wqflask/wqflask/__init__.py

```diff
@@ -19,6 +24,11 @@
 
 # Registering blueprints
 app.register_blueprint(glossary_blueprint, url_prefix="/glossary")
+app.register_blueprint(references_blueprint,url_prefix="/references")
+app.register_blueprint(links_blueprint,url_prefix="/links")
+app.register_blueprint(policies_blueprint,url_prefix="/policies")
+app.register_blueprint(environments_blueprint,url_prefix="/environments")
+app.register_blueprint(facilities_blueprint,url_prefix="/facilities")
```

Comment:
**nitpick:** Add a space between the "...blueprint,url_.prefix.."

We'll have to add a linter to catch these things for us. Nice to keep things consistent. Makes things easy on the eyes, and maintenance less effortful :)